cli/upload.py

```diff
@@ -225,12 +236,16 @@ def _compose_file_mapping(upload_info: api.UploadInfo, xlsx: str):
     res = []
     xlsx_dir = os.path.abspath(os.path.dirname(xlsx))
     for local_path, gcs_uri in upload_info.url_mapping.items():
```

Comment:
should we rename `local_path` to `source_path` here, since it might be a path to a remote file in GCS?